cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/legalrep/ContactDetails.java

```diff
@@ -1,15 +1,16 @@
-package uk.gov.hmcts.cmc.claimstore.models.legalrep;
+package uk.gov.hmcts.cmc.domain.models.legalrep;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 import org.hibernate.validator.constraints.Email;
-import uk.gov.hmcts.cmc.claimstore.constraints.PhoneNumber;
+import uk.gov.hmcts.cmc.domain.models.constraints.PhoneNumber;
+import uk.gov.hmcts.cmc.domain.utils.ToStringStyle;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.gov.hmcts.cmc.domain.utils.ToStringStyle'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)